ci-operator/config/cri-o/cri-o/cri-o-cri-o-main__ci.yaml

```diff
@@ -6,26 +11,41 @@ build_root:
     tag: golang-1.20
 images:
 - dockerfile_literal: |
-    FROM registry.fedoraproject.org/fedora:38
+    FROM base
+    RUN dnf install -y jq git openssl python3.11 && \
+        ln -sf /usr/bin/python3.11 /usr/bin/python3 && \
+        dnf clean all
     RUN curl https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz > /tmp/google-cloud-sdk.tar.gz
+    RUN python3 --version
     RUN mkdir -p /usr/local/gcloud \
       && tar -C /usr/local/gcloud -xvf /tmp/google-cloud-sdk.tar.gz \
-      && /usr/local/gcloud/google-cloud-sdk/install.sh \
       && rm -f /tmp/google-cloud-sdk.tar.gz
     ENV PATH $PATH:/usr/local/gcloud/google-cloud-sdk/bin
     WORKDIR /src
     COPY . .
-  from: src
+  from: base
   to: crio-crio-base-src
+- dockerfile_literal: |
+    FROM base
+    RUN dnf install -y jq git openssl python3.11 google-cloud-cli-447.0.0-1 gzip jq unzip openssh-clients openssl bind-utils util-linux xz && \
```

Comment:
Follow-up: Can we use the latest python 3 and gcloud cli here? I fear that those versions may get outdated in the future and we don’t see it.

Reply:
Yeah, that's a good point.

Reply:
Let me get the CI working, and then I'll update all dependencies to the latest version.